share/spice/holiday/holiday.js

```diff
@@ -0,0 +1,117 @@
+(function (env) {
+	"use strict";
+	env.ddg_spice_holiday = function(api_result) {
+	    if(!api_result || !api_result.h || api_result.h.length === 0) {
+			return Spice.failed('holiday');
```

Comment:
looks like we have some tabs mixed in here. Sadly, I'll need you to remove them ;)